common/views/components/ImageViewer/ImageViewerImage.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import {Component} from 'react';
-import {convertImageUri, convertIiifUriToInfoUri} from '@weco/common/utils/convert-image-uri';
-import {spacing} from '@weco/common/utils/classnames';
+import {convertImageUri, convertIiifUriToInfoUri} from '../../../utils/convert-image-uri';
+import {spacing} from '../../../utils/classnames';
 import openseadragon from 'openseadragon';
```

Comment:
We should have `openseadragon` in `common/pacakge.json` too no?

Reply:
hmmmmm… probably – will investigate

Reply:
We do: https://github.com/wellcometrust/wellcomecollection.org/blob/cardigan-work-media/common/package.json#L30